packages/manager/modules/anthos/src/dashboard/host/set-state/translations/Messages_fr_FR.json

```
@@ -0,0 +1,8 @@
+{
+  "anthos_dashboard_set_state_host_title": "Définir l’état",
+  "anthos_dashboard_set_state_host_description": "Par défaut, tous les hosts sont considérés comme stateless . En cas d'incident, OVHcloud pourra tenter une réinstallation ou changement de machine, après en avoir demandé l'évacuation à Kubernetes. Vous pouvez marquer certains hosts comme « stateful ». En cas d'incident, nous vous contacterons et mettrons des efforts en oeuvre pour conserver les données locales des hots stateful pendant la maintenance, dans la limite du possible.",
```

```suggestion
  "anthos_dashboard_set_state_host_description": "Par défaut, tous les hosts sont considérés comme stateless. En cas d'incident, OVHcloud pourra tenter une réinstallation ou changement de machine, après en avoir demandé l'évacuation à Kubernetes. Vous pouvez marquer certains hosts comme « stateful ». En cas d'incident, nous vous contacterons et mettrons des efforts en œuvre pour conserver les données locales des hots stateful pendant la maintenance, dans la limite du possible.",
```

> (...)  des hots stateful pendant la maintenance, (...)

Shouldn't be `hosts` instead of `hots`?